### PR TITLE
Feature/improve UI

### PR DESCRIPTION
**Pull request by chaoran-chen**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -30,7 +30,9 @@
             "styles": [
               "src/scss/styles.scss"
             ],
-            "scripts": []
+            "scripts": [
```
Comment (vordemann) [inline angular.json:33]: Did not install marked automatically when I ran ng serve.

File: src/app/header/header.component.html [hunk 2]
```diff
@@ -13,13 +13,16 @@
     <div class="header-nav">
       <ul class="header-list">
         <li class="header-list-item">
-          <a class="header-link" routerLink="/network">Discover</a>
+          <a class="header-link" routerLink="/">
+            <img src="../../assets/img/iglu.png" class="home-icon" />
```
Comment (vordemann) [inline src/app/header/header.component.html:17]: The iglu looks more like an eye and it does not look good in the navigation bar next to "Home". Would argue for removing it again.

File: src/app/device-search/device-search.component.html [hunk 3]
```diff
@@ -1,39 +1,54 @@
 <div id="main">
   <div class="container-fluid content">
-    <div class="row justify-content-center">
-      <div class="col-lg-12">
-        <h2 class="text-center">Devices</h2>
+    <div class="row justify-content-center header">
+      <div class="col-lg-7">
+        <h2>Devices</h2>
+        <p>Are you looking for a specific device and/or someone who is familiar with it? Explore our latest devices!</p>
```
Comment (vordemann) [inline src/app/device-search/device-search.component.html]: Use of word device is too often.
Rephrase: ... Explore our latest entries!

**Review by vordemann — CHANGES_REQUESTED** [review]

File: angular.json (hunk 1 above)
Comment (chaoran-chen) [reply angular.json:33]: Did you run `npm install`? We have `ngx-markdown` in our package.json which again depends on `marked`.

File: src/app/header/header.component.html (hunk 2 above)
Comment (chaoran-chen) [reply src/app/header/header.component.html:17]: I actually like it. Let's discuss it tomorrow?

File: src/app/device-search/device-search.component.html (hunk 3 above)
Comment (chaoran-chen) [reply src/app/device-search/device-search.component.html]: Good point! I changed it.

File: src/app/header/header.component.html (hunk 2 above)
Comment (vordemann) [reply src/app/header/header.component.html:17]: ok

File: angular.json (hunk 1 above)
Comment (vordemann) [reply angular.json:33]: Yes, i did install it with npm

**Review by vordemann — APPROVED** [review]

**Pull request merged**